packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -250,15 +261,30 @@ export default class GLTFScene extends React.Component<Props, {| loadedModel: ?O
   }
 
   render() {
+    const { children, ...rest } = this.props;
+
     const { loadedModel } = this.state;
     if (!loadedModel) {
       return null;
     }
+
+    const drawHitmap =
+      children.id && (rest.onDoubleClick || rest.onMouseDown || rest.onMouseUp || rest.onMouseMove || rest.onClick);
+    const sharedDrawProps = { model: loadedModel, ...children };
+
     return (
       <WorldviewReactContext.Consumer>
         {(context) => {
           this._context = context;
-          return <Command reglCommand={drawModel} drawProps={{ model: loadedModel, ...this.props.children }} />;
+          return (
+            <Command
+              {...rest}
+              reglCommand={drawModel}
+              drawProps={sharedDrawProps}
+              hitmapProps={drawHitmap ? { ...sharedDrawProps, drawHitmap } : undefined}
```

Comment:
Seems strange that you tell the command about hitmap in 2 different ways: by passing `hitmapProps` _and_ by including a `drawHitmap` key inside the hitmap props. Only one of these should be necessary right? The command renderer can decide what to do based on whether there is an id.

Having to use `(ctx, props) => !!props.x` instead of `regl.prop("x")` is a bit of a code smell too; if you expect a boolean then the user should pass in a boolean.